packages/docs/package.json

```diff
@@ -26,7 +26,7 @@
   "dependencies": {
     "@blueprintjs/core": "^1.34.0",
     "classnames": "^2.2",
-    "documentalist": "^0.0.8",
+    "documentalist": "^0.1.1",
```

Comment:
use `1.0.0-beta.0` for testing and `1.0.0` for the final thing that gets merged into this repo. no point in a `0.1.x` version range